src/App.tsx

```diff
@@ -1,8 +1,10 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
+import * as facilityService from "../src/services/FacilityService";
 
 function App() {
+  const data = facilityService.getAllFacilities();
```

Comment:
what is this?

Reply:
this was just for testing. It will be remove later.